conf/solr/7.3.1/schema.xml

```diff
@@ -0,0 +1,1639 @@
+<?xml version="1.0" encoding="UTF-8" ?>
```

Comment:
Is this identical to the 7.3.0 schema?

Reply:
Good catch - they are identical, and the 7.3.0 directory should be deleted in this PR.